src/api/operator/operatorApi.ts

```diff
@@ -106,13 +106,7 @@ export async function getTxOrders(params: GetTxOrdersParams): Promise<RawOrder[]
 
   console.log(`[getTxOrders] Fetching tx orders on network ${networkId}`)
 
-  // sdk not merging array responses yet
-  const orders = await Promise.all([
-    COW_SDK.cowApi.getTxOrders(txHash, { chainId: networkId, env: 'prod' }),
-    COW_SDK.cowApi.getTxOrders(txHash, { chainId: networkId, env: 'staging' }),
-  ])
-
-  return [...orders[0], ...orders[1]]
+  return orderBookSDK(networkId, environmentName === 'production' ? 'prod' : 'staging').getTxOrders(txHash)
```

Comment:
Thanks for the comment! I wanted to discuss it.
What's the reason for it? Looks like in most cases we make redundant requests and load on the backend.

Reply:
p.s. i moved it back, just curious

Reply:
Because the explorer is meant to be a view on the protocol, of what happens on and off chain.

We have a database segregation on the backend, while the underlying smart contract is the same

If we rely only on a single env, it can happen that on the landing page we show the link to the latest batch
![image](https://user-images.githubusercontent.com/43217/218981715-2ece9ef0-3de2-4d8c-9a1d-cff369b63e7c.png)

But when you open it's empty as you are viewing the prod endpoint but the batch was settled on barn and vice versa